backend/common/models.py

```diff
@@ -87,26 +88,29 @@ class Payment(BaseModel):
     status: PaymentStatus = PaymentStatus.NOT_STARTED
 
 
-class RequestStatus(Enum):
+class RequestStatus(str, Enum):
     PENDING = "PENDING"
     ACCEPTED = "ACCEPTED"
     DECLINED = "DECLINED"
 
 
-class RequestType(Enum):
+class RequestType(str, Enum):
     JOIN_GROUP = "JOIN_GROUP"
     JOIN_MINIGAME = "JOIN_MINIGAME"
 
 
 class Request(BaseModel):
-    id: str
-    group_id: str
-    sender_id: str
-    recipient_id: str
-    date: datetime
-    type: RequestType
+    id: Optional[PydanticObjectId] = Field(alias="_id", default=None)
+    group_id: PydanticObjectId
+    sender_id: PydanticObjectId
+    recipient_id: PydanticObjectId
+    date: datetime = datetime.now()
```

Comment:
The date should be populated at object initialization. When written so, when the app runs, `datetime.now()` is executed and every object will have the same date value